Lecture3.tex

```diff
@@ -153,7 +153,7 @@ \subsection{The Bellman-Ford Algorithm}
 
       \tab{if  $d[v] < \infty $}
 
-      \tab{\tab{set $\pi [v] \in \arg {\min _u}\{ w(v,u) + \tilde d[u]:(v,u) \in E\} $}}
+      \tab{\tab{set $\pi [v] \in \arg {\min _u}\{ w(v,u) + d[u]:(v,u) \in E\} $}}
```

Comment:
This is actually OK as it is

Reply:
@avivt I agree - I restored the notation